app/src/main/java/dev/shreyaspatil/foodium/ui/main/MainActivity.kt

```diff
@@ -166,13 +166,13 @@ class MainActivity : BaseActivity<MainViewModel, ActivityMainBinding>(),
 
     override fun onBackPressed() {
         MaterialDialog.Builder(this)
-            .setTitle("Exit?")
-            .setMessage("Are you sure want to exit?")
-            .setPositiveButton("Yes") { dialogInterface, _ ->
+            .setTitle(getString(R.string.exit_dialog_title))
+            .setMessage(getString(R.string.exit_dialog_message))
+            .setPositiveButton(getString(R.string.yes)) { dialogInterface, _ ->
                 dialogInterface.dismiss()
                 super.onBackPressed()
             }
-            .setNegativeButton("No") { dialogInterface, _ ->
+            .setNegativeButton(getString(R.string.no)) { dialogInterface, _ ->
```

Comment:
Instead of `yes` or `no` can you please make it like
`option_yes` and `option_no`.

Reply:
@GeorgCantor Thanks for your contribution. But please read contribution guidelines before direct making a PR.